### ISSUE TITLE
Doesn't allows Object parameter description, using multiple params

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#84]
```
What steps will reproduce the problem?
1. Create function with valid jsdoc, containing object description with 
multiple params, like this: 
https://gist.github.com/anonymous/e57415453d755e5e6e9f
2. Run closure linter.
3. Receive error about jsdoc description for missing parameter.

What is the expected output? What do you see instead?

Expected to see no error. Receive error.

What version of the product are you using? On what operating system?

Latest version. Linux/MacOS.
```

Original issue reported on code.google.com by `ymi...@gmail.com` on 28 Aug 2014 at 2:33
